nix/shell.nix

```diff
@@ -7,8 +7,15 @@ in
 with pkgs;
 mkShell {
   buildInputs = [
-    pkgs.bazel-buildtools
-    pkgs.python3
+    bazel-buildtools
+    python3
+    rust-bin.nightly."2023-12-01".default
+    # cargo
+    rust-bindgen
+    # rustfmt
+    patchelf
+    nodejs
+    # custom
```

Comment:
Should `cargo`, `rustfmt`, or `custom` be comments?

Reply:
Sorry. Let me remove "custom" was meant to be a descriptive comments; I'll delete the others.